sklearn/tests/test_kernel_approximation.py

```diff
@@ -118,6 +118,20 @@ def test_skewed_chi2_sampler():
     assert_raises(ValueError, transform.transform, Y_neg)
 
 
+def test_additive_chi2_sampler_exceptions():
+    """Ensures correct error message"""
+    transform = AdditiveChi2Sampler()
```

Comment:
maybe name it `transformer` or `estimator` or `est` because transform is a bit confusing